### PR TITLE
change reg authority search to use all ocds at level

### PR DESCRIPTION
**Pull request by tank157**
With this in place, if we fold in AKA ocd-ids (so we have multiple at the same level), we can try them all and take any of the found authorities (presumably there'll only be one though)

### CODE REVIEW

File: test/voter_registration_works/states_test.clj [hunk 1]
```diff
@@ -25,43 +25,80 @@
...
+                               {:voter-registration-authority-level :county
+                                :voter-registration-methods
+                                #{{:type :by-mail}}}]}
+        test-county-authority {:types #{:election :registration}
+                               :ocd-division
+                               [county-ocd-id "City and County of Denver"]
+                               :foo "bar"}]
+    (testing "returns reg. authority for by-mail registration"
+      (let [test-voter {:register-address :test
+                    :language-preference :es
```
Comment (cap10morgan) [inline test/voter_registration_works/states_test.clj]: Looks like the indentation got a little wonky here.

**Comment by cap10morgan** [comment]
Some indentation cleanup needed in the tests, but o/w LGTM!

**Comment by cap10morgan** [comment]
@tank157 Is it time to get this merged?

**Comment by tank157** [comment]
@cap10morgan I'm not sure. It seemed like the thing to do when I wrote it, but later on I seemed to have second thoughts for some reason. I'm trying to jog my memory if there was some reason I didn't merge it, other than I thought the whole matching system needed an overhaul.

**Comment by cap10morgan** [comment]
@tank157 In the meantime should we close the PR but leave the branch?

**Comment by tank157** [comment]
@cap10morgan That seems reasonable

**Pull request closed without merging**